synapse/lib/cell.py

```diff
@@ -1467,7 +1470,6 @@ async def initNexusSubsystem(self):
         if self.cellparent is None:
             await self.nexsroot.recover()
             await self.nexsroot.startup()
-            await self.setCellActive(self.conf.get('mirror') is None)
```

Comment:
I vaguely recall it being important that the cell is determined to be active or not immediately after starting up nexus. I think it might not be ok for nexus to be online but our active state to be ambiguous?

Reply:
*sigh* good to know. I'll put it back. That's going to complicate things a lot.